src/components/MetricsPane/MetricsPane.styles.js

```diff
@@ -21,7 +21,7 @@ export const StyledMetricsWrapper = styled.div`
     z-index: 5;
     background-color: transparent;
     width: 90%;
-    bottom: ${(props) => (props.$showLoadingIndicator ? '5rem;' : '0.5rem;')}
```

Comment:
I think these conditional styles based on the loading indicator were vestigial. I cant see them impacting anything visually anymore and removing this helped me move the state back into the loading component instead of in a parent (that is already doing a lot)